src/utils/transformationDescription.ts

```diff
@@ -104,6 +106,32 @@ export function useActiveTransformations(
     async function callback(deletedContext: string) {
       const cloned = { ...activeTransformations };
       for (const input of Object.keys(cloned)) {
+        // Partition transformations based on whether or not one of their input
+        // contexts was deleted
+        const transformationsWithNewlyMissingInputs = [];
+        const restOfTransformations = [];
+        for (const description of activeTransformations[input]) {
+          if (
+            description.inputs.includes(deletedContext) ||
+            description.extraDependencies.includes(deletedContext)
+          ) {
+            transformationsWithNewlyMissingInputs.push(description);
+          } else {
+            restOfTransformations.push(description);
+          }
+        }
+
+        // Rename transformations with newly missing inputs to add a [fixed] suffix
+        for (const transformation of transformationsWithNewlyMissingInputs) {
+          if ("output" in transformation) {
+            console.log("transformation", transformation);
+            const outputData = await getDataContext(transformation.output);
+            await updateDataContext(transformation.output, {
+              title: `${outputData.title} [fixed]`,
+            });
```

Comment:
It'd be nice to also update the description here (maybe just appending to the end) to give a more verbose explanation of what the "[fixed]" label means for this table.

Reply:
Unfortunately, the API does not allow updating the description for some reason. I've reproduced this in the API tester: if you send an "update data context" request with a `metadata: {description: "test"}` field, then the request succeeds but the description isn't actually updated. I can look into this more but thankfully I don't think it's a deal-breaking issue.

Reply:
Ahhh yes I remember this--Jonathan confirmed that you can't currently update it through the API [here](https://codap.concord.org/forums/topic/accessing-dataset-description-through-plugin-api/). 

I guess this has implications for our ideas about flowing name changes since dataset names appear in descriptions and from that point on we just can't update them.